boot/boot_serial/src/boot_serial.c

```diff
@@ -666,6 +666,9 @@ boot_serial_start(const struct boot_uart_funcs *f)
 
     off = 0;
     while (1) {
+#if defined(CONFIG_LOG) && !defined(CONFIG_LOG_IMMEDIATE)
+        k_sleep(3); /* give time to process the log */
```

Comment:
Magic value `3` should have a define with some comment on rationale. Otherwise it is extremely hard for future developers to understand the implications of it, and how to maintain it on other HW (if needed)

Reply:
Lol, I was doing per commit review, and I see now that this is indeed removed all together :)